recipes/libpcsclite/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set name = "libpcsclite" %}
+{% set version = "1.9.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pcsclite.apdu.fr/files/pcsc-lite-{{ version }}.tar.bz2
+  sha256: cbcc3b34c61f53291cecc0d831423c94d437b188eb2b97b7febc08de1c914e8a
+
+build:
+  skip: True  # [not linux]
+  script:
+    - set -e
+    - set -x
+    - if test -f Makefile; then make distclean; fi
+    - rm -rf *.cache *.m4 config.guess config.log config.status config.sub depcomp ltmain.sh
+    - autoreconf --verbose --install --force
+    - ./configure --prefix=$PREFIX --sbindir=$PREFIX/bin
+    - make install
+    - make distcheck
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('libpcsclite') }}
```

Most of the time ABI compatability does not sync with the API version. Do you have any documentation that they only break the ABI at major API releases?

Ask upstream whether they only do ABI breaks at certain API releases.

```suggestion
    - {{ pin_subpackage('libpcsclite', max_pin='x.x.x') }}
```

---

@leofang What do  you think?

---

@carterbox i think it'd be too strict, see my earlier comment: https://github.com/conda-forge/staged-recipes/pull/21405#discussion_r1054838829 that said I would not mind if you wanna be on the cautious side

---

btw

> Most of the time ABI compatability does not sync with the API version.

I am not sure this is true for traditional Linux packages...

---

> @carterbox i think it'd be too strict, see my earlier comment: [#21405 (comment)](https://github.com/conda-forge/staged-recipes/pull/21405#discussion_r1054838829) that said I would not mind if you wanna be on the cautious side

All of the samples on ABI lab are v1.8, so there is not enough evidence to say whether they would break the ABI at a minor API change. I think the breaking ABI changes from 1.8.9 to 1.8.10 were an accident. I still recommend asking upstream, but pinning to `'x.x'` is probably? safe.

---

Sounds good to me!

---

```suggestion
    - {{ pin_subpackage('libpcsclite', max_pin='x.x') }}
```